sources/input_to_text_tree.c

```diff
@@ -6,12 +6,13 @@
 /*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/09/14 23:07:09 by inikulin          #+#    #+#             */
-/*   Updated: 2024/11/04 05:58:42 by taretiuk         ###   ########.fr       */
+/*   Updated: 2024/12/02 19:59:18 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 #ifdef MOCK_TANIA
+# include "tokenizing/input_processing.h"
```

Comment:
please move include to lower section

Reply:
i put it between minishell.h and MOCK_TANIA, ok?